src/Views/IframeContentView.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Handle form template view skin.
+ *
+ * @package Give
+ */
+
+namespace Give\Views;
+
+/**
+ * Class IframeView
+ *
+ * @since 2.7.0
+ * @package Give
+ */
+class IframeContentView {
+	/**
+	 * Document page title.
+	 *
+	 * This will be use to setup title tag.
+	 *
+	 * @since 2.7.0
+	 * @var string
+	 */
+	protected $title;
+
+	/**
+	 * Document page body.
+	 *
+	 * This will be use to setup content of body tag.
+	 *
+	 * @since 2.7.0
+	 * @var string
+	 */
+	protected $body;
+
+	/**
+	 * Body classes.
+	 *
+	 * This will be use to setup body tag classes.
+	 *
+	 * @since 2.7.0
+	 * @var array
+	 */
+	protected $bodyClasses = [ 'give-form-templates' ];
+
+	/**
+	 * Set document page title.
+	 *
+	 * @param string $title
+	 *
+	 * @return IframeContentView $this
+	 */
+	public function setTitle( $title ) {
+		$this->title = $title;
+
+		return $this;
+	}
+
+	/**
+	 * Set document page title.
+	 *
+	 * @param string $body
+	 *
+	 * @return IframeContentView $this
+	 */
+	public function setBody( $body ) {
+		$this->body = $body;
+
+		return $this;
+	}
+
+	/**
+	 * Set document page title.
+	 *
+	 * @param array $classes
+	 *
+	 * @return IframeContentView $this
+	 */
+	public function setBodyClasses( $classes ) {
+		$this->bodyClasses = array_merge( $this->bodyClasses, (array) $classes );
+
+		return $this;
+	}
+
+
+	/**
+	 * Render view.
+	 *
+	 * Note: if you want to overwrite this function then do not forget to add action hook in footer and header.
+	 * We use these hooks to manipulated donation form related actions.
+	 *
+	 * @since 2.7.0
+	 */
+	public function render() {
+		ob_start();
+		?>
+		<!DOCTYPE html>
+		<html <?php language_attributes(); ?>>
+			<head>
+				<meta charset="utf-8">
+				<title><?php echo apply_filters( 'the_title', $this->title ); ?></title>
```

Comment:
Needs to be escaped

Reply:
@bradp Class property is a source of truth, so I believe that the developer will pass an escaped string. I will handle this where we are calling this function